app/classifier/classification-summary.jsx

```diff
@@ -36,6 +36,14 @@ class ClassificationSummary extends React.Component {
     return false;
   }
 
+  showExoplanetSimFeedback() {
+    return (this.props.workflow.configuration &&
+      this.props.workflow.configuration.sim_notification &&
+      this.isSubjectASim() &&
+      this.props.classification.annotations[0].value === 0
```

Comment:
In that case, I'm OK to leave this as is. There's no logical way to reach this section of the code unless the project is Exoplanets Explorer, and if EE always enforces an answer, then we won't need to worry about nulls.